sdk/test/trace/tracer_test.cc

```diff
@@ -341,6 +341,28 @@ TEST(Tracer, SpanSetAttribute)
   ASSERT_EQ(3.1, nostd::get<double>(span_data->GetAttributes().at("abc")));
 }
 
+TEST(Tracer, SpanSetEvents)
+{
+  std::shared_ptr<std::vector<std::unique_ptr<SpanData>>> spans_received(
+      new std::vector<std::unique_ptr<SpanData>>);
+
+  auto tracer = initTracer(spans_received);
+
+  auto span = tracer->StartSpan("span 1");
+  span->AddEvent("event 1");
+  span->AddEvent("event 2", std::chrono::system_clock::now());
```

Comment:
Can we test adding a third event with attributes?

Reply:
Thanks! Can we actually test that some attributes and up in the recordable? Like `span->AddEvent("event 3", {{"attr1", 1}})`?